sympy/printing/ccode.py

```diff
@@ -394,37 +516,128 @@ def _print_NegativeInfinity(self, expr):
+        return self._print_math_func(expr)
 
-    def _print_hypot(self, expr):
-        return '{0}hypot({1}, {2})'.format(self._ns, *map(self._print, expr.args))
+    def _print_exp2(self, expr):
+        return self._print_math_func(expr)
 
     def _print_expm1(self, expr):
-        return '{0}expm1({1})'.format(self._ns, self._print(expr.args[0]))
+        return self._print_math_func(expr)
 
-    def _print_log1p(self, expr):
-        return '{0}log1p({1})'.format(self._ns, self._print(expr.args[0]))
+    def _print_log(self, expr):
+        return self._print_math_func(expr)
 
-    def _print_exp2(self, expr):
-        return '{0}exp2({1})'.format(self._ns, self._print(expr.args[0]))
+    def _print_log10(self, expr):
+        return self._print_math_func(expr)
 
     def _print_log2(self, expr):
-        return '{0}log2({1})'.format(self._ns, self._print(expr.args[0]))
+        return self._print_math_func(expr)
+
+    def _print_log1p(self, expr):
+        return self._print_math_func(expr)
+
+    def _print_Cbrt(self, expr):
+        return self._print_math_func(expr)
+
+    def _print_hypot(self, expr):
+        return self._print_math_func(expr)
+
+    def _print_fma(self, expr):  # fused mutiply-add
+        return self._print_math_func(expr)
 
     def _print_loggamma(self, expr):
-        return '{0}lgamma({1})'.format(self._ns, self._print(expr.args[0]))
+        return self._print_math_func(expr)
 
-    # tgamma was already covered by 'known_functions' dict
+    def _print_sin(self, expr):
+        return self._print_math_func(expr)
+
+    def _print_cos(self, expr):
+        return self._print_math_func(expr)
+
+    def _print_tan(self, expr):
+        return self._print_math_func(expr)
+
+    def _print_asin(self, expr):
+        return self._print_math_func(expr)
+
+    def _print_acos(self, expr):
+        return self._print_math_func(expr)
+
+    def _print_atan(self, expr):
+        return self._print_math_func(expr)
+
+    def _print_atan2(self, expr):
+        return self._print_math_func(expr)
+
+    def _print_sinh(self, expr):
+        return self._print_math_func(expr)
+
+    def _print_cosh(self, expr):
+        return self._print_math_func(expr)
+
+    def _print_tanh(self, expr):
+        return self._print_math_func(expr)
+
+    def _print_asinh(self, expr):
+        return self._print_math_func(expr)
+
+    def _print_acosh(self, expr):
+        return self._print_math_func(expr)
+
+    def _print_atanh(self, expr):
+        return self._print_math_func(expr)
+
+    def _print_erf(self, expr):
+        return self._print_math_func(expr)
+
+    def _print_erfc(self, expr):
+        return self._print_math_func(expr)
+
+    def _print_loggamma(self, expr):
+        return self._print_math_func(expr)
+
+    def _print_gamma(self, expr):
+        return self._print_math_func(expr)
+
+    def _print_ceiling(self, expr):
+        return self._print_math_func(expr)
+
+    def _print_floor(self, expr):
```

Comment:
You can just assign these all: `_print_floor = _print_ceiling = ... = _print_math_func`.